src/tasks/tasks.py

```diff
@@ -740,24 +721,18 @@ def get_metrics(self, reset=False):
                 'precision': pcs, 'recall': rcl}
 
 
-@register_task('qqp-alt', rel_path='QQP/')
-class QQPAltTask(QQPTask):
```

Comment:
From the other PR thread: (Alex) I think the original motivation for structuring the -alt tasks is that you could have attention for the -alt, but not the original task. Does this setup still allow us to do that? I think so because task-specific module construction depends on task name and not task class, but seems worth testing to be sure.

Reply:
Confirmed that if you use settings like `sts-b-pair_attn = 0, sts-b-alt-pair_attn = 1`, you will create a model with two different components, one of which has attention and one of which will not. Looks good!